docs/text/forecasting.rst

```diff
@@ -26,290 +31,203 @@ The following image illustrates the process:
-+----+------+----+----+
-| 2  | t9   | 11 | 13 |
-+----+------+----+----+
-
-If you set rolling to 1, the feature extraction works with all of the following time series:
+.. code:: python
 
-+----+------+----+----+
-| id | time | x  | y  |
-+====+======+====+====+
-| 1  | t1   | 1  | 5  |
-+----+------+----+----+
-
-+----+------+----+----+
-| id | time | x  | y  |
-+====+======+====+====+
-| 1  | t1   | 1  | 5  |
-+----+------+----+----+
-| 1  | t2   | 2  | 6  |
-+----+------+----+----+
-
-+----+------+----+----+
-| id | time | x  | y  |
-+====+======+====+====+
-| 1  | t1   | 1  | 5  |
-+----+------+----+----+
-| 1  | t2   | 2  | 6  |
-+----+------+----+----+
-| 1  | t3   | 3  | 7  |
-+----+------+----+----+
-| 2  | t8   | 10 | 12 |
-+----+------+----+----+
-
-+----+------+----+----+
-| id | time | x  | y  |
-+====+======+====+====+
-| 1  | t1   | 1  | 5  |
-+----+------+----+----+
-| 1  | t2   | 2  | 6  |
-+----+------+----+----+
-| 1  | t3   | 3  | 7  |
-+----+------+----+----+
-| 1  | t4   | 4  | 8  |
-+----+------+----+----+
-| 2  | t8   | 10 | 12 |
-+----+------+----+----+
-| 2  | t9   | 11 | 13 |
-+----+------+----+----+
-
-If you set rolling to -1, you end up with features for the time series, rolled in the other direction
-
-+----+------+----+----+
-| id | time | x  | y  |
-+====+======+====+====+
-| 1  | t4   | 4  | 8  |
-+----+------+----+----+
-
-+----+------+----+----+
-| id | time | x  | y  |
-+====+======+====+====+
-| 1  | t3   | 3  | 7  |
-+----+------+----+----+
-| 1  | t4   | 4  | 8  |
-+----+------+----+----+
-
-+----+------+----+----+
-| id | time | x  | y  |
-+====+======+====+====+
-| 1  | t2   | 2  | 6  |
-+----+------+----+----+
-| 1  | t3   | 3  | 7  |
-+----+------+----+----+
-| 1  | t4   | 4  | 8  |
-+----+------+----+----+
-| 2  | t9   | 11 | 13 |
-+----+------+----+----+
-
-+----+------+----+----+
-| id | time | x  | y  |
-+====+======+====+====+
-| 1  | t1   | 1  | 5  |
-+----+------+----+----+
-| 1  | t2   | 2  | 6  |
-+----+------+----+----+
-| 1  | t3   | 3  | 7  |
-+----+------+----+----+
-| 1  | t4   | 4  | 8  |
-+----+------+----+----+
-| 2  | t8   | 10 | 12 |
-+----+------+----+----+
-| 2  | t9   | 11 | 13 |
-+----+------+----+----+
-
-We only gave an example for the flat DataFrame format, but rolling actually works on all 3 :ref:`data-formats-label`
-that are supported by tsfresh.
+   import pandas as pd
+   df = pd.DataFrame({
+      "id": [1, 1, 1, 1, 2, 2],
+      "time": [1, 2, 3, 4, 8, 9],
+      "x": [1, 2, 3, 4, 10, 11],
+      "y": [5, 6, 7, 8, 12, 13],
+   })
 
+Now, we can use :func:`tsfresh.utilities.dataframe_functions.roll_time_series` to get consecutive sub-time series.
+You could think of having a window sliding over your time series data and extracting out every data you can see through this window.
+There are three parameters to tune the window:
+
+* `rolling_direction`: if you want to slide in positive (increasing sort) or negative (decreasing sort) direction. Default is positive.
```

Comment:
That is actually a good point. I basically kept it for consistency (and somehow backward compatibility). In principle, foreward and backward rolling does the same, it is just a matter of nomenclature (and what happens on the borders). I would keep the functionality, just because we always had it. But honestly, I also do not see a lot of use cases...